<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPatientDialog</class>
 <widget class="QDialog" name="AddPatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пациенты - Добавление</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayoutDataForm">
     <item row="4" column="0">
      <widget class="QLabel" name="labelPhone">
       <property name="text">
        <string>Номер телефона</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBoxAge">
       <property name="minimum">
        <number>18</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelFullname">
       <property name="text">
        <string>ФИО Пациента</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelAge">
       <property name="text">
        <string>Возраст</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelDistrict">
       <property name="text">
        <string>Район</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditFullname"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditRegion"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditPhone"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelRegion">
       <property name="text">
        <string>Регион</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxDistrict"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutAction">
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
